www/source/partials/docs/_glossary-keys.html.md.erb

```diff
@@ -1,14 +1,29 @@
 # <a name="glossary-keys" id="glossary-keys" data-magellan-target="glossary-keys">Keys</a>
 
-Habitat has strong cryptography built into both Habitat Builder and the Supervisor. This means there are several different kinds of keys.
+Habitat has strong cryptography built into Habitat Builder, the Supervisor, and the `hab` CLI commands. This means there are several different kinds of keys.
 
-## Origin Keys
+## Origin Key Pairs
 
-As described in the [packages documentation](/docs/glossary/#glossary-packages), every package in Habitat belongs to an origin and is cryptographically signed with that origin's private key.
+Every Habitat artifact belongs to an [origin](/docs/glossary/#glossary-artifacts) and is cryptographically signed with that origin's private key.
+Habitat requires both the public and the private key for producing and consuming artifacts.
```

Comment:
This makes it sound like both halves of the key pair are required for both operations. How about
```suggestion
Habitat requires the private key for producing artifacts and the public key and consuming them.
```
?